src/components/PageNotFound/PageNotFound.scss

```
@@ -0,0 +1,26 @@
+.pagenotfound-container {
+  width: 100%;
+  padding: 20px;
+
+  &__head{
+    color: #1b1b1b;
```

No need to introduce new colors
Probably wont be needed after you reuse markdown selector
```suggestion
    color: getColor(fiord);
```